tests/core/test_validation_definition.py

```diff
@@ -214,6 +236,10 @@ def test_adds_requisite_ids(
         assert output.meta == {
             "validation_id": validation_definition.id,
             "checkpoint_id": checkpoint_id,
+            "batch_spec": ACTIVE_BATCH_SPEC,
```

Comment:
It looks like this test uses a `mock_validator`. Does this just insert these values? Do we need a test where we run a real validation and verify we see these fields?

Reply:
the test in `test_checkpoint.py` is a full integration test that runs a real validation. i could break it out into its own test, if that's clearer.